### PR TITLE
Adding age prevalence curves

### PR DESCRIPTION
**Pull request by dtcitron**
Added a new function for plotting age prevalence curves

Also changed the plotting functions to return the ggplot object, so that we can still interact with them or edit them afterwards.

### CODE REVIEW

File: R/PlotEMODResults.R [hunk 1]
```diff
@@ -70,9 +70,10 @@ emodplot.prevalence <- function(data,
     mutate(Prevalence = case_when(Population == 0 ~ 0,
                                   Population > 0 ~ Infected / Population))
   y.lim.max <- min(max(data$Prevalence) * 1.2, 1)
-  emodplot.by_gender(data, date.start, date.end, 'Prevalence', title=title ) +
+  p <- emodplot.by_gender(data, date.start, date.end, 'Prevalence', title=title ) +
     scale_y_continuous(labels = scales::percent_format(accuracy = 1), breaks = seq(0, y.lim.max,0.05), limits=c(0,y.lim.max)) +
     ylab("HIV Prevalence (%)")
+  return(p)
```
Comment (kaftand) [inline R/PlotEMODResults.R:76]: This is great. I don't think it changes behavior - an R function should return the output of the last line in the function. However, this is much easier to read.

Comment (dtcitron) [reply R/PlotEMODResults.R:76]: I was having trouble catching the plot object outside of the function before? Not sure what was going on, but this should fix it.

**Comment by dtcitron** [comment]
I updated the commit to make a more generalized way of plotting age-prevalence curves, as well as a function for explicitly plotting age and prevalence.

File: R/PlotEMODResults.R [hunk 2]
```diff
@@ -38,7 +38,7 @@ emodplot.by_gender <- function(data,
     # geom_errorbar(data = prevalence.data, aes(x=Year, ymin=lb*1, ymax=ub*1), color="black", width=2, size=1) +
     facet_wrap(~ Gender, ncol=2) +
     xlab("Year")+
-    xlim(c(date.start, date.end)) +
+    #xlim(c(date.start, date.end)) +
```
Comment (kaftand) [inline R/PlotEMODResults.R:41]: Was this comment intentional? Otherwise, this looks good to go.

**Review by kaftand — CHANGES_REQUESTED** [review]
Everything looks good except perhaps the `xlim` being commented in `emodplot.by_gender`

**Pull request merged**